Starforge.Vanilla/Tools/TileRectangleTool.cs

```diff
@@ -73,5 +78,9 @@ public class TileRectangleTool : Tool {
             Hint.Width = 8;
             Hint.Height = 8;
         }
+
+        public override void RenderGUI() {
+            TileBrushTool.CreateGUITileTool();
```

Comment:
This seems a little weird, maybe it would be a better idea to make an abstract general "TileTool" that implements the CreateGUI and that other Tile related tools (TileRectangle and TileBrush) can inherit from?